front/app/containers/Admin/settings/statuses/components/IdeaStatusForm.tsx

```diff
@@ -100,6 +107,13 @@ const IdeaStatusForm = ({ defaultValues, onSubmit }: Props) => {
     }
   };
 
+  const codes: TIdeaStatusCode[] = [
```

Comment:
I think there are some issues here. When there are multiple statuses with the same code, the code appears multiple times in the radio buttons. When we delete all statuses with a specific code, then the code does not appear and a status with that code cannot be reintroduced.

Perhaps we could use two hard coded lists? Is there a way to access the variant from here?